spec/support/corpus.rb

```diff
@@ -124,6 +116,40 @@ def checkout
 
     private
 
+      MissingExpectedException = Class.new(StandardError)
+
+      # Count mutations and check error results against whitelist
+      #
+      # @param path [Pathname] path responsible for exception
+      #
+      # @return [Fixnum] mutations generated
+      def count_mutations_and_check_errors(path)
+        relative_path = path.relative_path_from(repo_path)
+
+        count = count_mutations(path)
+
+        expected_errors.assert_success(relative_path)
```

Comment:
@dkubb I added `@raise` to [line 144](https://github.com/mbj/mutant/pull/566/files#diff-a6b9f047fe088f086dce98b7ec90f402R144). It is pretty generic because the method generically raises an error from either `parser` or `regexp_parser` but it conveys this